google-sheets/functions/check-sheets-config.js

```diff
@@ -0,0 +1,48 @@
+const { google } = require('googleapis');
+const path = require('path');
+
+exports.handler = async function(context, _event, callback) {
+    const response = new Twilio.Response();
+    response.appendHeader('Content-Type', 'application/json');
+
+    try {
+        const authJson = require(path.join('..', context.SHEETS_AUTH_JSON));
```

Comment:
This won't work because when Functions and Assets are deployed their names get rewritten. So instead you'll have to do:

```suggestion
        const authJson = require(Runtime.getAssets()[context.SHEETS_AUTH_JSON].path);
```

But `Runtime.getAssets()` will expect `context.SHEETS_AUTH_JSON` to be the URL path of a private asset. Hence it has to be `/auth.json`.

Reply:
This makes sense! I ended up changing this to an `fs.readFile` on the Asset, because Node.JS `require` doesn't see Asset paths as JSON files for deployed Functions (likely the `.json` extension gets stripped when not running locally).